core/common/scale/util.hpp

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_SCALE_UTIL_HPP
+#define KAGOME_SCALE_UTIL_HPP
+
+#include <algorithm>
+#include <cstdint>
+#include <vector>
+
+#include <boost/endian/arithmetic.hpp>
+
```

Comment:
```suggestion
```

Reply:
This is going to be removed as well as boost/predef.h